tests/components/rest/test_sensor.py

```diff
@@ -876,7 +876,7 @@ async def test_update_with_xml_convert_bad_xml(
     respx.get("http://localhost").respond(
         status_code=HTTPStatus.OK,
         headers={"content-type": "text/xml"},
-        content="",
+        content="<open></close>",
```

Comment:
Instead of changing the test, parametrize it:
```py
@pytest.mark.parametrize(("content", "error_message"), [("", "Empty reply"), ("<open></close>", "Erroneous JSON")])
```

Reply:
Thanks for the hint, done